content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -781,10 +781,10 @@ crontab "my-new-cron-object" created
 
 ### Defaulting
 
-{{< feature-state state="beta" for_kubernetes_version="1.16" >}}
+{{< feature-state state="stable" for_kubernetes_version="1.17" >}}
 
 {{< note >}}
-Defaulting is available as beta since 1.16 in `apiextensions.k8s.io/v1` CustomResourceDefinitions, and hence enabled by default for most clusters (feature gate `CustomResourceDefaulting`, refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation).
+Defaulting is available as beta since 1.16 in `apiextensions.k8s.io/v1` CustomResourceDefinitions, and hence enabled by default for most 1.16 clusters (feature gate `CustomResourceDefaulting`, refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation).
```

Comment:
```suggestion
To use defaulting, your CustomResourceDefinition must use API version `apiextensions.k8s.io/v1`.
```
IMO, the feature-state shortcode above makes it clear what's supported in current Kubernetes (v1.17 will be current once this is merged), and the hyperlink to feature gates lets any reader work out what's available in previous versions. The docs also link to previous docs releases.